pkgs/applications/version-management/datalad/default.nix

```diff
@@ -1,68 +1,86 @@
-{ lib, stdenv, fetchFromGitHub, installShellFiles, python3, git, git-annex }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  installShellFiles,
+  python3,
+  git,
+  git-annex,
+  p7zip,
+  curl,
+}:
 
 python3.pkgs.buildPythonApplication rec {
   pname = "datalad";
-  version = "1.0.2";
+  version = "1.1.1";
 
   src = fetchFromGitHub {
     owner = "datalad";
     repo = pname;
     rev = "refs/tags/${version}";
-    hash = "sha256-oq+DdlWcwjJSQdnqHlYCa9I7iSOKf+hI35Lcv/GM24c=";
+    hash = "sha256-Vw/RpMf+jnUijJ3GZ9nLk1IRWOADmM+jNtYl5Ba6uLg=";
   };
 
-  nativeBuildInputs = [ installShellFiles git ];
-
-  propagatedBuildInputs = with python3.pkgs; [
-    # core
-    platformdirs
-    chardet
-    iso8601
-    humanize
-    fasteners
-    packaging
-    patool
-    tqdm
-    annexremote
-    looseversion
-    setuptools
-    git-annex
-
-    # downloaders-extra
-    # requests-ftp # not in nixpkgs yet
-
-    # downloaders
-    boto
-    keyrings-alt
-    keyring
-    msgpack
-    requests
-
-    # publish
-    python-gitlab
-
-    # misc
-    argcomplete
-    pyperclip
-    python-dateutil
-
-    # metadata
-    simplejson
-    whoosh
-
-    # metadata-extra
-    pyyaml
-    mutagen
-    exifread
-    python-xmp-toolkit
-    pillow
-
-    # duecredit
-    duecredit
-
-    # python>=3.8
-    distro
-  ] ++ lib.optionals stdenv.hostPlatform.isWindows [ colorama ]
+  nativeBuildInputs = [
+    installShellFiles
+    git
+  ];
+
+  build-system = [ python3.pkgs.setuptools ];
+
+  dependencies =
```

Comment:
I think we need to switch dependencies. They've changed them a lot: https://github.com/datalad/datalad/blob/0b18b32ee05cf6d7d12772719458ae5526d52553/setup.py#L53

Reply:
They did change a bit. I adjusted it (removed  `simplejson` and `whoosh`).

Reply:
Yes, but for example pyyaml, mutagen and others do not exist in `setup.py`. You need to remove them.